.circleci/config.yml

```diff
@@ -198,6 +183,9 @@ workflows:
           filters:
             branches:
               only: /^pull\/.*/
+          matrix:
+            parameters:
+              python: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
you wouldn't happen to know a way to keep this in sync with [python versions in tox](https://github.com/Sceptre/sceptre/blob/master/tox.ini#L3) would you?

Reply:
Agreed that would be nice, but didn't find a simple solution after some searching. Seems like you could use https://circleci.com/docs/dynamic-config/ to generate the parameters from the tox.ini file

Reply:
as commented above, run unit tests thru tox instead of pytest?

Reply:
For now I think it's easier to just define the list of Python versions in two places instead of trying to figure out how to generate one from the other.